### PR TITLE
Design

### PR DESCRIPTION
**Pull request by Sabotnik**
I think I resolved the merge conflicts - let's pull from this going forward. I went through it with my mentor today.

### CODE REVIEW

**Comment by Matt-Lepley** [comment]
I pulled this, and it looks good except for the background image doesn't load. I don't think it was passed through in assets? Should we just use a direct link to the image instead of an asset?

**Comment by Matt-Lepley** [comment]
Also, now that I look at it, I think we need a register button as well

**Comment by Sabotnik** [comment]
@donsalvadori can you swap in a direct link instead of an asset for the background? 
@Matt-Lepley what do you mean by a register button? - thought that was the 'sign in'

**Comment by Matt-Lepley** [comment]
@Sabotnik Sign In refers to signing in if you already have an account created. Register/Signup would be to create a new account.

File: app/models/game.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 class Game < ActiveRecord::Base
+<<<<<<< HEAD
```
Comment (kenmazaika) [inline app/models/game.rb]: Looks like there are still some gotchas in here about merge conflicts.  We should fix these up before we merge.

**Comment by Matt-Lepley** [comment]
Merging this.. still need to add register link, fix link paths, and change image path from relative to absolute.

**Comment by Matt-Lepley** [comment]
Merging this.. still need to add register link, fix link paths, and change image path from relative to absolute.

**Pull request merged**